[ENHANCEMENT] Update error boundary answer (#161)
30-seconds/30-seconds-of-interviews: questions/error-boundaries.md

Error boundaries are React components that catch JavaScript errors anywhere in their child component tree, log those errors, and display a fallback UI instead of the component tree that crashed.

>  class component becomes an error boundary if it defines either (or both) of the lifecycle methods `static getDerivedStateFromError()` or `componentDidCatch().`

Reviewer: Small grammar fixes

```suggestion
Class components become error boundaries if they define either (or both) of the lifecycle methods `static getDerivedStateFromError()` or `componentDidCatch().`
```

#### Good to hear

> * <b>Error boundaries only catch errors in the components below them in the tree.</b> An error boundary can’t catch an error within itself.

Reviewer: Please use Markdown and not HTML markup
```suggestion
* **Error boundaries only catch errors in the components below them in the tree.** An error boundary can’t catch an error within itself.
```
